packages/eas-cli/src/commands/workflow/create.ts

```diff
@@ -0,0 +1,107 @@
+import chalk from 'chalk';
+import fs from 'fs/promises';
+import path from 'path';
+import prompts from 'prompts';
+
+import EasCommand from '../../commandUtils/EasCommand';
+import Log from '../../log';
+
+const WORKFLOWS_DIR = '.eas/workflows';
+const DEFAULT_WORKFLOW_NAME = 'workflow.yml';
+const HELLO_WORLD_TEMPLATE = `name: Hello World
+
+on:
+  push:
+    branches: ['*']
+
+jobs:
+  Hello World:
+    steps:
+      - run: echo "Hello, World"
+`;
+
+export class WorkflowCreate extends EasCommand {
+  static override description = 'create a new workflow configuration YAML file';
+
+  static override args = [
+    {
+      name: 'name',
+      description: 'Name of the workflow file (must end with .yml or .yaml)',
+      required: false,
+    },
+  ];
+
```

Comment:
```suggestion
  static override contextDefinition = {
    ...this.ContextOptions.ProjectDir,
  }
```
Let's add this context here. It will ensure that this command can be only run inside of a project directory vs some other random place. I believe we also need to use this context in the `workflow:validate` command.